api/turing/service/ensembling_job_service.go

```diff
@@ -21,8 +21,8 @@ type EnsemblingJobFindByIDOptions struct {
 // EnsemblingJobListOptions holds query parameters for EnsemblersService.List method.
 type EnsemblingJobListOptions struct {
 	PaginationOptions
-	ProjectID *models.ID
-	Status    *models.Status `validate:"oneof=pending running terminating terminated completed failed failed_submission failed_building"` //nolint
+	ProjectID *models.ID `schema:"project_id" validate:"required"`
+	Status    *models.Status
```

Comment:
Why has the `validate` tag been removed?

Reply:
It was removed because now it is required if you want to filter by status, which I did not intend. Anyway in my other PR I have changed this to accept a list of statuses instead.

Reply:
So, should `Status` be removed altogether from this PR?

Reply:
Err, there's some unit tests referencing this so I don't think it's worth the effort.

Reply:
Anyway in my other PR I will add in the querying multiple statuses, seems like a logical flow as well.